packages/flutter/lib/src/material/radio_list_tile.dart

```diff
@@ -488,11 +488,17 @@ class RadioListTile<T> extends StatelessWidget {
     switch (controlAffinity) {
       case ListTileControlAffinity.leading:
       case ListTileControlAffinity.platform:
-        leading = control;
+        leading = Focus(
```

Comment:
This could go before the switch:

```dart
 control = Focus(
            descendantsAreFocusable: false,
            canRequestFocus: false,
            child: control);
```